usr.sbin/mountd/mountd.c

```diff
@@ -196,6 +196,7 @@ struct fhreturn {
 	int	*fhr_secflavors;
 };
 
+static char error_buf[PATH_MAX];
```

Comment:
If we include path in the message, then it would have sense to provide some more space for the error itself.